recipes/ckan-editor-utils/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "ckan-editor-utils" %}
+{% set version = "0.1.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fca949890bf319f62d33f693f2bf4a04dc28a66e70d4d1b38173bff46bad2302
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry >=1.1.3
+  run:
+    - boto3 >=1.15.16,<2.0.0
+    - python >=3.6,<4.0
+    - requests >=2.24.0,<3.0.0
+
+test:
+  imports:
+    - ckan_editor_utils
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/ckan-editor-utils/
+  summary: Utilities for editing CKAN using its API.
```

```suggestion
  summary: Utilities for editing CKAN, the Comprehensive Knowledge Archive Network
```

Just a suggestion. I had no idea what CKAN was and I find it nice to see acronyms expanded when I go to feedstocks without having to google them.